Applications/Help/main.cpp

```diff
@@ -159,10 +165,21 @@ int main(int argc, char* argv[])
         model->update_section_node_on_toggle(index, open);
     };
 
+    auto open_external = [](auto& url) {
+        if (!Desktop::Launcher::open(url))
+            dbg() << "Failed to open '" << url << "'";
+    };
+
     page_view.on_link_click = [&](auto& url, auto&, unsigned) {
-        char* current_path = strdup(history.current().characters());
-        char* path = realpath(url.path().characters(), nullptr);
```

Comment:
Yay for getting rid of the `realpath()` call! Hopefully one day we'll be able to make `Core::File::real_path_for()` not rely on `realpath()` (and `PATH_MAX`) internally :smile: